lib/tools.js

```diff
@@ -1571,6 +1571,6 @@ module.exports = {
     encrypt,
     decrypt,
     ERRORS: {
-        ERROR_NOT_EXISTS: 'Not exists'
+        ERROR_NOT_FOUND: 'Not exists'

```

Comment:
@GermanBluefox can we change the actual error message text or does some code depend on exactly `'Not exists'`?

Reply:
Many, many, many different places...... in Other adapters